lib/media/playhead.js

```diff
@@ -427,7 +427,9 @@ shaka.media.MediaSourcePlayhead = class {
         // infinite loop on systems where changing currentTime takes a
         // significant amount of time (e.g. Chromecast).
         const time = Date.now() / 1000;
-        if (!this.lastCorrectiveSeek_ || this.lastCorrectiveSeek_ < time - 1) {
+        const seekSlowDelay = shaka.util.Platform.isFuchsia() ? 3 : 1;
```

Comment:
```suggestion
        const seekSlowDelay = shaka.util.Platform.isFuchsiaCastDevice() ? 3 : 1;
```

Reply:
Well it was not needed due to the is seek slow that already catch cast device but i update it
it will help if we have any refactor